dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.9636.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_BUILD_TYPE="Release"
+LLVM_MAX_SLOT="13"
+MY_PN="igc"
+MY_P="${MY_PN}-${PV}"
+#PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_10 )
```

Comment:
Why Python 3.10 only?

Reply:
I do not know if 3.10 is really necessary, but cmake checks for 3.10 and find it (python 3.10 is stable so I have installed). 
I put it rather for remark of more investigations about if we really need 3.10.

Reply:
cmake will always find (and compile with) only the latest enabled python target unless you put the cmake stuff inside a call to `python_foreach_impl cmake-stuff.....`. 

If python is only required during build time (which is implied by the usage of `python-any-1`) then this behaviour is correct and you can (and should) just expand PYTHON_COMPAT.

If python is also required during runtime the ebuild should use `python-single-r1` or `python-r1` in combination with `python_foreach_impl`.